chihacknight/il-solar-map
# added updates section and 2 posts

## Overview

To better highlight updates to this site and to document data updates, created a Updates section of the site for short blog-style updates. This PR also adds 2 posts - a data update from Sept and publishing the Top 10.

### Demo

![Screenshot 2024-10-22 at 2 03 00 PM](https://github.com/user-attachments/assets/0801813f-6dfe-4475-906c-2a447c2563ca)

![Screenshot 2024-10-22 at 2 03 08 PM](https://github.com/user-attachments/assets/54c04a30-7b04-4a36-ab9f-5257f4d6da57)

## Testing Instructions

* New updates section show show up
* make sure links work
* check sharing updates on social media
